repository/f.json

```diff
@@ -521,6 +521,18 @@
 				}
 			]
 		},
+		{
+			"name": "File Filter",
+			"details": "https://github.com/vsc-cnst/SublimeTextFileFilter",
+			"author": "vsc",
+			"labels": [ "file filter", "file content filter", "logs filter", "lines filter", "file", "line", "filter", "regex" ],
```

Comment:
The author is fine, but wouldn't you prefer that to be simply your GitHub username?

The labels won't help that might I think... they're just all random combinations of file and filter etc. Reads as an attempt to hack the SEO :D None of them will mean much to users. Just use a few of the regularly used ones, e.g. from this list: https://packagecontrol.io/browse/labels